flytectl/cmd/compile/compile.go

```diff
@@ -105,6 +100,65 @@ func compileFromPackage(packagePath string) error {
 	return nil
 }
 
+func handleWorkflow(
+	workflow *admin.WorkflowSpec,
+	compiledTasks []*core.CompiledTask,
+	compiledWorkflows map[string]*core.CompiledWorkflowClosure,
+	compiledLaunchPlanProviders []common.InterfaceProvider,
+	plans map[string]*admin.LaunchPlan,
+	workflows map[string]*admin.WorkflowSpec) ([]common.InterfaceProvider, error) {
+	reqs, _ := compiler.GetRequirements(workflow.Template, workflow.SubWorkflows)
+	wfName := workflow.Template.Id.Name
+
+	// Check if all the subworkflows referenced by launchplan are compiled
+	for i := range reqs.GetRequiredLaunchPlanIds() {
+		lpID := &reqs.GetRequiredLaunchPlanIds()[i]
+		lpWfName := plans[lpID.Name].Spec.WorkflowId.Name
+		missingWorkflow := workflows[lpWfName]
```

Comment:
In what sense is it missing? Would `currentWorkflow` also be a fitting name in your opinion?